onnxruntime/core/mlas/inc/mlas_gemm_postprocessor.h

```diff
@@ -0,0 +1,33 @@
+/*++
```

Comment:
why a new header file?

Reply:
this class is used in both mlas_q4.h and mlas_qnbit.h. we could potentially unify those header files later and move this definition back to that header.

Reply:
regarding the new header file mlas_qnbit.h, I thought it was a more appropriate place for the new n-bit quantized int Gemm function even if only 4-bit is supported now. open to other ideas for how to organize the headers though.